gcp/appengine/frontend3/src/templates/list.html

```diff
@@ -133,6 +135,13 @@ <h1 class="title">Vulnerability Library</h1>
               <span class="tag fix-unavailable">No fix available</span>
               {%- endif -%}
             </span>
+            <span role="cell" class="vuln-table-cell mdc-data-table__cell hide-on-mobile">
+              <ul>
+                {% for score in (vulnerability.severity_scores) %}
+                <li>{{ score }}</li>
+                {% endfor %}
```

Comment:
I think it would be preferable to only output the highest available CVSS score, I.e. where CVSSv3 and CVSSv4 scores are provided, favour the v4 score.